### PR TITLE
Update to 2020

### PR DESCRIPTION
**Pull request by e9t**
Resolves https://github.com/nlposs/NLP-OSS/issues/26

Sorry for the late updates!
I didn't manage to find a good design template for the page yet, but I thought displaying the contents ASAP would be more important than showing a pretty page.

Your feedback is welcome!

## Changes

- Copied the raw html of the 2018 webpage, and migrated to `./2018`
- Created an `./2020/index.md`, which contains major contents from [the proposal](https://github.com/nlposs/NLP-OSS/)
- Created an `./index.md` which redirects the page to `./2020`. This will allow easier expansions to future versions of NLP-OSS.

## What's missing

- ~A design template for the 2020 page~
- ~Appropriate dates in the `Important dates` section~
- Updated Workshop program
- ~Updates on the final list of `Programme Committee`~

## Quick preview

image
image

### CODE REVIEW

**Comment by goodmami** [comment]
Thanks for your hard work!

Are we not concerned with breaking links, e.g., if someone visits the 2018 page at the old URL or linked to one of the static files? We could maintain redirects to the new `2018/` location, except for the top-level `index.md`, I think.

**Comment by e9t** [comment]
Thanks for the heads up @goodmami!
I manually added some redirection pages for the pdfs. Please let me know if I've missed something important.

**Comment by alvations** [comment]
Hmmm, I do agree with @goodmami that somehow we have to make the links to the 2018 site a little more explicit. Other than that the content and all looks good.

Thank you @e9t for all the hard work!! 

I like the one markdown page to keep everything  idea. I think it helps to keep things clean too =)

File: 2020/index.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+ - [Svitlana Vakulenko](svakulenk0.github.io), Vienna University of Economics and Business
+ - [Gaël Varoquaux](http://gael-varoquaux.info), INRIA
+ - [Moshe Wasserblat](), Intel Corporation
+ - [Marcos Zampieri](http://uni-koeln.de/~mzampie2/index.html),  University of Wolverhampton
+ - [Maarten van Gompel](https://proycon.anaproy.nl/), Radboud University
+
+
+## Workshop Structure
+
+To promote closer interaction, all accepted publications will be presented in the poster format and be required to give a 1 slide (90 seconds) introduction of their work at the beginning of the session. For the rest of the workshop, invited talks will be given by developers and maintainers of prominent NLP-OSS projects and we plan to close the workshop with a panel session with the invited speakers and the audience of the workshop.
```
Comment (alvations) [inline 2020/index.md]: Maybe add this after the workshop structure section:

```
## Previous Workshop

[**First Workshop for Natural Language Processing Open Source Software** (NLP-OSS 2018)](http://nlposs.github.io/2018) <br>
\[[Proceedings](https://www.aclweb.org/anthology/volumes/W18-25/)\]
```

File: 2020/index.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+## Important dates
+
+The 2nd NLP-OSS workshop will be co-located with the EMNLP 2020 conference.
+
+- Paper submission: TBD
+- Paper Reviews Starts: TBD
+- Paper Reviews Due: TBD
+- Notification of Acceptance: TBD
+- Camera-Ready Version: TBD
+- Workshop: TBD
```
Comment (alvations) [inline 2020/index.md]: Using [WMT20](http://www.statmt.org/wmt20/) dates a guide, how about these:

```
- Paper submission: 01 July 2020
- Paper Reviews Starts: 05 July 2020
- Paper Reviews Due: 05 August 2020
- Notification of Acceptance: 10 August 2020
- Camera-Ready Version: 25 August 2020
- Workshop: 11 or 12 November 2020
```

File: 2020/index.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+## Invited Speakers
+
+- [Chip Huyen](https://huyenchip.com), NVIDIA
```
Comment (alvations) [inline 2020/index.md]: Chip has moved away from NVIDIA, got to find out what's her affiliation later on.

File: 2020/index.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+- [Spencer Kelly](https://spencermounta.in/), Freelance Developer
```
Comment (alvations) [inline 2020/index.md]: BTW, Spencer Kelly created yet another beautiful NLP tool, https://github.com/spencermountain/compromise-align 

Maybe we should ask him where he gets inspiration of the UI and the tools development from =)

File: 2020/index.md (hunk 2 above)
Comment (e9t) [reply 2020/index.md]: LGTM! Will apply first, and we can keep the discussion going until we merge this PR.

File: 2020/index.md (hunk 3 above)
Comment (e9t) [reply 2020/index.md]: Will put in a placeholder for now.

**Comment by e9t** [comment]
Also applied a very [simple jekyll theme](https://github.com/essentialenemy/noir):

image

**Comment by e9t** [comment]
Will merge after the internal discussion among organizers at Slack.

**Pull request merged**